contrib/build_sysimg.jl

```diff
@@ -161,9 +161,15 @@ function link_sysimg(sysimg_path=nothing, cc=find_system_compiler(), debug=false
         push!(FLAGS, "-lssp")
     end
 
+    sysimg_file = "$sysimg_path.$(Libdl.dlext)"
     info("Linking sys.$(Libdl.dlext)")
-    run(`$cc $FLAGS -o $sysimg_path.$(Libdl.dlext) $sysimg_path.o`)
-
+    if is_windows() && isfile(sysimg_file)
+        mv(sysimg_file, "$sysimg_file.old"; remove_destination=true)
+        run(`$cc $FLAGS -o $sysimg_file $sysimg_path.o`)
+        run(`rm "$sysimg_path.$(Libdl.dlext).old"`)
```

Comment:
Maybe. I have seen many times in windows that a stubborn file that refuses to be deleted just needs a rename, before you can delete it, strange but it work.